scripts/linters/general_purpose_linter.py

```diff
@@ -116,7 +116,9 @@ class BadStringsConstantsDict(TypedDict):
 BAD_PATTERNS: Dict[str, BadPatternsDict] = {
     '\t': {
         'message': 'Please use spaces instead of tabs.',
-        'excluded_files': (),
+        'excluded_files': (
+            'Makefile',
```

Comment:
Is it mandatory to use tabs and not spaces in the Makefile?

Reply:
yes, otherwise make command wont work